library/src/scala/tasty/reflect/SymbolOps.scala

```diff
@@ -82,6 +82,9 @@ trait SymbolOps extends Core {
     def isAnonymousFunction(implicit ctx: Context): Boolean = kernel.Symbol_isAnonymousFunction(self)
     def isAbstractType(implicit ctx: Context): Boolean = kernel.Symbol_isAbstractType(self)
     def isClassConstructor(implicit ctx: Context): Boolean = kernel.Symbol_isClassConstructor(self)
+
+    def isType(implicit ctx: Context): Boolean = kernel.Symbol_isType(self)
+    def isTerm(implicit ctx: Context): Boolean = kernel.Symbol_isTerm(self)
```

Comment:
If you really want to keep this method, it could be implemented as `kernel.matchesTermSymbol(self).isDefined`. Similar for `def isType`.